source/CoordinateConversion/CoordinateConversionLibrary/Helpers/ImportCSV.cs

```diff
@@ -38,33 +39,48 @@ public object Convert(List<string> row, string dateformat = "")
 
     public class ImportCSV
     {
-        public static List<string> GetHeaders(Stream stream, char seperator)
+        public static List<string> GetHeaders(Stream stream)
         {
             using (StreamReader reader = new StreamReader(stream))
             {
                 string line = reader.ReadLine();
                 if (string.IsNullOrEmpty(line))
                     return null;
 
-                return line.Split(seperator).ToList();
+                return line.Split(GetSeperator(line)).ToList();
```

Comment:
Just a very minor thing - "separator" is misspelled throughout PR (looks like it was misspelled on original line above so that probably instigated it)